src/voice.ts

```diff
@@ -44,12 +41,14 @@ export class Voice extends Element {
   protected readonly tickables: Note[];
   protected readonly time: VoiceTime;
 
-  // Modes allow the addition of ticks in three different ways:
-  //
-  // STRICT: This is the default. Ticks must fill the voice.
-  // SOFT:   Ticks can be added without restrictions.
-  // FULL:   Ticks do not need to fill the voice, but can't exceed the maximum
-  //         tick length.
+  /**
+   * Modes allow the addition of ticks in three different ways:
+   *
+   * STRICT: This is the default. Ticks must fill the voice.<br>
```

Comment:
Is the \<br\> tag the only way to force a line break? Is the alternative to have an extra empty line between the STRICT line and SOFT line?

Personally I would prefer having as few HTML tags in the comments as possible, but I could live with the br tags if there is no good alternative.

What about putting the three different ways in a bulleted list?

```
Modes allow the addition of ticks in three different ways:

- STRICT: This is the default. Ticks must fill the voice.
- SOFT:   Ticks can be added without restrictions.
- FULL:   Ticks do not need to fill the voice, but can't exceed the maximum tick length.
```

Reply:
Done